migrations/README.md

```diff
@@ -78,10 +94,10 @@ database with a given group of work, write a migration.
 - Use the generator `rails g data_migration some_new_data_migration`.
 - Make them reversible in case we have to rollback.
 
-### Temporary Rake Task
+#### Temporary Rake Task
 
 - We use these if a migration should not be tied to a deploy. Examples:
-  - Long running migrations.
+  - Long running migrations that don't need to be interwoven.
```

Comment:
Could you talk more about the "interwoven" language of this line? I'd like to understand this part :-)

Reply:
Much more definition around that and the "why" of the different things.